tests/qos/test_qos_sai.py

```diff
@@ -366,6 +366,8 @@ def testQosSaiBufferPoolWatermark(
                 RunAnsibleModuleFail if ptf test fails
         """
         disableTest = request.config.getoption("--disable_test")
+        if dutTestParams["basicParams"]["sonic_asic_type"] == 'cisco-8000':
+            disableTest = False
```

Comment:
--disable_test is by default set to True which is causing to skip this testcase by default.
The intention of above code is to avoid skipping on cisco-8000